### PR TITLE
Fix french translation for compare validator

### PR DESCRIPTION
**Pull request by gael-wogenstahl**
Compare validator error messages translation are incorrect in french.
`{compareValue}` is used instead of `{compareValueOrAttribute}`

### CODE REVIEW

File: framework/messages/fr/yii.php [hunk 1]
```diff
@@ -82,17 +82,17 @@
     '{attribute} must be an IP address with specified subnet.' => '{attribute} doit être une adresse IP avec un sous-réseau.',
     '{attribute} must be an integer.' => '{attribute} doit être un entier.',
     '{attribute} must be either "{true}" or "{false}".' => '{attribute} doit être soit « {true} » soit « {false} ».',
-    '{attribute} must be greater than "{compareValue}".' => '{attribute} doit être supérieur à « {compareValue} ».',
-    '{attribute} must be greater than or equal to "{compareValue}".' => '{attribute} doit être supérieur ou égal à « {compareValue} ».',
-    '{attribute} must be less than "{compareValue}".' => '{attribute} doit être inférieur à « {compareValue} ».',
-    '{attribute} must be less than or equal to "{compareValue}".' => '{attribute} doit être inférieur ou égal à « {compareValue} ».',
+    '{attribute} must be greater than "{compareValueOrAttribute}".' => '{attribute} doit être supérieur à « {compareValueOrAttribute} ».',
```
Comment (SilverFire) [inline framework/messages/fr/yii.php]: There is a redundant space before `»` in all translations

Comment (samdark) [inline framework/messages/fr/yii.php]: Yes https://en.wikipedia.org/wiki/Guillemet

**Comment by samdark** [comment]
Thanks!

**Comment by gael-wogenstahl** [comment]
:+1:

**Pull request merged**